src/main/java/gift/main/service/KakaoService.java

```diff
@@ -0,0 +1,56 @@
+package gift.main.service;
+
+import gift.main.config.KakaoProperties;
+import gift.main.dto.KakaoProfile;
+import gift.main.dto.KakaoTokenResponse;
+import gift.main.dto.UserJoinRequest;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestClient;
+
+@Service
+public class KakaoService {
+
+    private static MediaType CONTENT_TYPE = MediaType.valueOf("application/x-www-form-urlencoded;charset=utf-8");
```

Comment:
이미 정의된 값을 활용해서 사용해볼 수 있을 것 같아요.
```suggestion
private static final MediaType CONTENT_TYPE = new MediaType(APPLICATION_FORM_URLENCODED, UTF_8);
```

Reply:
헉 이런게 있었군요.. 뭔가 있을법한데 없다면 제가 못찾은 것이라고 생각하고 더 찾아보는 습관을 가져야할 것 같습니다...
덕분에 코드가 더 깔끔해졌습니다 감사합니다!

Reply:
반영했습니다 ^0^